src/main/java/network/radicle/jetbrains/radiclejetbrainsplugin/patches/PatchProposalPanel.java

```diff
@@ -225,7 +225,7 @@ protected SimpleChangesBrowser getChangesBrowser(Project project, PatchTabContro
 
     protected void calculatePatchChanges(CountDownLatch latch) {
         ApplicationManager.getApplication().executeOnPooledThread(() -> {
-            // TODO fetch first and then show the changes
+            // TODO fetch first and then show the changes, maybe i have to add the did to the remotes
```

Comment:
I think this should be moved as a new issue

Reply:
Yes, I didn't want to make the changes to this branch. Too much changes